static/app/views/issueDetails/groupRelatedIssues/index.spec.tsx

```diff
@@ -0,0 +1,87 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {RouterFixture} from 'sentry-fixture/routerFixture';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {GroupRelatedIssues} from 'sentry/views/issueDetails/groupRelatedIssues';
+
+describe('Related Issues View', function () {
+  let relatedIssuesMock: jest.Mock;
+  let issuesInfoMock: jest.Mock;
+  const router = RouterFixture();
+
+  const organization = OrganizationFixture();
+  const orgSlug = organization.slug;
+  const groupId = '12345678';
+  const group1 = '15';
+  const group2 = '20';
+  // query=issue.id:[15,20] -> query=issue.id%3A%5B15%2C20%5D
```

Comment:
Some commented out code here

Reply:
It's on purpose to help understand the string on the next line.